docs/setting-up/server/easy-install.md

```diff
@@ -17,7 +17,7 @@ wget -O - https://www.percona.com/get/pmm | /bin/bash
 This script:
 
 - Installs Docker if it is not already installed on your system.
-- Stops and backs up any PMM Server Docker containers that are currently running.
+- Stops and renames any PMM Server Docker container found to be running (not a recoverable backup being taken)
```

Comment:
```suggestion
- Stops and renames any PMM Server Docker container currently running from `pmm-server` to `pmm-server-backup-{timestamp}`. This new pmm-server container based on the latest version is not a recoverable backup.
```

Reply:
instead of 

"This new pmm-server container based on the latest version is not a recoverable backup.",

Can it be "This old pmm-server container is not a recoverable backup."?